### ISSUE TITLE
the ; keyboard key does not work

### ISSUE THREAD

**Issue by nbarkhina** [caiiiycuk/js-dos.com#48]
Is there any way to set custom keyboard mappings? this is an important key because you can't navigate to C:\ or ; in any code editors like QBasic. I've tried in both Chrome and Edge and neither browser registers the key. does anyone else have this issue?

Thanks

**Comment by caiiiycuk** [comment]
Yes I know this issue. I will fix it, when I have time (hope on this week). Btw I think you can try to use [mapper file](https://www.dosbox.com/wiki/Mapper). And start js-dos with this config options:
```
main(['-startmapper', '-conf', 'dosbox.conf'])
```

And in dosbox.conf you should put path to mapper file like this:
```
[sdl]
mapperfile=mapper.map
```

Please write any feedback you have, it will help me to solve this issue.

**Comment by nbarkhina** [comment]
Thank you! I will try the mapper file and see if that works

**Comment by nbarkhina** [comment]
I was able to get the custom DOSBox.conf file loading but I don't have a good mapper file. where is the one you are using? Maybe I could just try modifying that. I see it's trying to load mapper-jsdos.map but I don't see that in your source.

**Comment by caiiiycuk** [comment]
I am sorry for long delay, it's because I am overloaded now) Anyway `;` works only in Firefox. This special keys `:;` some others have different keycodes in different browsers. To fix this, I need to build map of keycodes for each browser and then remap it to single standard codes. I suggest to use FF if you can, while I am fixing this bug.

**Comment by Oele** [comment]
Just wanted to add that apart from `;`, also the `'` key is not being registered. This one does not work in Firefox.

**Comment by nbarkhina** [comment]
Ok thanks for the update

**Comment by caiiiycuk** [comment]
Now should be fixed (6.22.18), if you find another key that doesn't work please open new issue.

**Comment by nbarkhina** [comment]
Thanks!! I will test it out
